tools/nebula-spark/src/main/java/com/vesoft/nebula/bean/ConnectInfo.java

```diff
@@ -38,13 +34,4 @@ public String getSpaceName() {
         }
         return hostAndPortList;
     }
-
-    @Override
-    public void check() throws IllegalArgumentException {
-        Preconditions.checkArgument(StringUtils.isNotEmpty(spaceName),
```

Comment:
maybe we can move the check stage into constructor

Reply:
The check stage has been moved to NebulaDataSource.createRelation，if it's better to check the parameters in ConnectInfo's constructor and ScanInfo's constructor?